scalapact-scalatest/src/main/scala/com/itv/scalapact/ScalaPactMock.scala

```diff
@@ -25,33 +26,35 @@ object ScalaPactMock {
     var port = -1
 
     try {
-      socket.setReuseAddress(true)
+      socket.setReuseAddress(false)
       port = socket.getLocalPort
 
       try {
         socket.close()
       } catch {
         // Ignore IOException on close()
-        case e: IOException =>
+        case _: IOException =>
       }
     } catch{
-      case e: IOException =>
+      case _: IOException =>
     } finally {
       if (socket != null) {
         try {
           socket.close()
         } catch {
-          case e: IOException =>
+          case _: IOException =>
         }
```

Comment:
Not specific to the PR, But what's going on with the same try/catch happening 3 times in a row? Whats happening here?